backend/compound_soaking/src/tables/soak_compound.rs

```diff
@@ -0,0 +1,38 @@
+use async_graphql::SimpleObject;
+use chrono::{DateTime, Utc};
+use sea_orm::{
+    ActiveModelBehavior, DeriveEntityModel, DerivePrimaryKey, DeriveRelation, EntityTrait,
+    EnumIter, PrimaryKeyTrait,
+};
+use uuid::Uuid;
+
+/// Represents a soaked compound within the database.
+#[derive(Clone, Debug, DeriveEntityModel, PartialEq, SimpleObject)]
+#[sea_orm(table_name = "soaked_compounds")]
+#[graphql(name = "soaked_compounds", complex)]
+pub struct Model {
+    /// ID of the plate on which the compound is located.
+    #[sea_orm(primary_key, auto_increment = false)]
+    pub compound_plate_id: Uuid,
+    /// The well on the plate which the compound is located.
+    #[sea_orm(primary_key, auto_increment = false)]
+    pub compound_well_number: i16,
+    /// ID of the plate on which the crystal is located.
+    #[sea_orm(primary_key, auto_increment = false)]
+    pub crystal_plate_id: Uuid,
+    /// The well on the plate which the crystal is located.
+    #[sea_orm(primary_key, auto_increment = false)]
+    pub crystal_well_number: i16,
+    /// The volume of compounds transferred into the crystal well.
+    pub volume: f32,
```

Comment:
What are your thoughts on datatype here? Why did you go with `f32`?

Reply:
I thought the compounds might be a few ml. Do we need more precision?

Reply:
I think they're probably smaller than that, maybe microlitres? Worth asking one of the lab staff. My thought was that if were going to be doing maths on these small numbers we're likely to come across some floating point errors, this could be avoided by using an integer type of specified resolution - e.g. u32 where 1 is 1ul